pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -266,7 +266,8 @@ func fitsRequest(podRequest *preFilterState, nodeInfo *framework.NodeInfo, ignor
 			nodeInfo.Allocatable.Memory,
 		})
 	}
-	if nodeInfo.Allocatable.EphemeralStorage < podRequest.EphemeralStorage+nodeInfo.Requested.EphemeralStorage {
+	if utilfeature.DefaultFeatureGate.Enabled(features.LocalStorageCapacityIsolation) &&
+		nodeInfo.Allocatable.EphemeralStorage < podRequest.EphemeralStorage+nodeInfo.Requested.EphemeralStorage {
```

Comment:
I am not sure I am following, if the feature is disabled, shouldn't `podRequest.EphemeralStorage+nodeInfo.Requested.EphemeralStorage` be always equal to 0? and if it is 0, then it will never be larger than `nodeInfo.Allocatable.EphemeralStorage`, right? what am I missing?

Reply:
@ahg-g you're right. This shouldn't be the root cause. I will dig out more.